### DISCUSSION TITLE
變聲器融入AI技術！ 讓你的聲音秒變「好萊塢巨星」

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#443] [General]
這種AI變聲都很好玩，看起來超好笑，很有趣~~
新聞網址:https://tw.news.yahoo.com/%E8%AE%8A%E8%81%B2%E5%99%A8%E8%9E%8D%E5%85%A5ai%E6%8A%80%E8%A1%93-%E8%AE%93%E4%BD%A0%E7%9A%84%E8%81%B2%E9%9F%B3%E7%A7%92%E8%AE%8A-%E5%A5%BD%E8%90%8A%E5%A1%A2%E5%B7%A8%E6%98%9F-000000194.html

**Comment by BingRong1217** [comment]
AI變聲近年來有很大的發展
